linkis-dist/package/db/upgrade/1.4.0_schema/mysql/linkis_ddl.sql

```diff
@@ -29,3 +29,5 @@ ALTER TABLE `linkis_ps_udf_user_load`  ADD COLUMN `update_time` datetime NOT NUL
 
 
 ALTER TABLE `linkis_ps_udf_version`  ADD COLUMN `update_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'update_time';
+
+ALTER TABLE `linkis_ps_udf_tree` ADD CONSTRAINT  `name_user_name_category` UNIQUE (`name`, `user_name`, `category`);
```

Comment:
need update linkis-dist/package/db/linkis_ddl.sql 
Follow the specification https://github.com/apache/linkis/blob/master/linkis-dist/package/db/linkis_ddl.sql
![image](https://user-images.githubusercontent.com/7869972/236868114-f483967b-2f5c-4fbd-b9ed-9d4303f4fde5.png)

Reply:
done